src/metabase/models/permissions.clj

```diff
@@ -451,6 +451,18 @@
   [perm-type perm-value db-id]
   (base->feature-perms-path perm-type perm-value (adhoc-native-query-path db-id)))
 
+(s/defn general-perms-path :- Path
+  "Get general permission's path by permission's type"
```

Comment:
You should make this a complete sentence https://guide.clojure.style/#docstring-summary

Reply:
Ok, I actually realized you just moved this from the other namespace but it's still worth fixing here